modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -1468,34 +1857,30 @@ Type::
 | `logTypes`
 | `string`
 | `logTypes` defines the desired record types to generate. Possible values are: +
- - `FLOWS` (default) to export regular network flows +
- - `CONVERSATIONS` to generate events for started conversations, ended conversations as well as periodic "tick" updates +
- - `ENDED_CONVERSATIONS` to generate only ended conversations events +
- - `ALL` to generate both network flows and all conversations events +
+ - `Flows` (default) to export regular network flows +
+ - `Conversations` to generate events for started conversations, ended conversations as well as periodic "tick" updates +
+ - `EndedConversations` to generate only ended conversations events +
+ - `All` to generate both network flows and all conversations events +
 
 
 | `metrics`
 | `object`
 | `Metrics` define the processor configuration regarding metrics
 
-| `port`
-| `integer`
-| Port of the flow collector (host port). By convention, some values are forbidden. It must be greater than 1024 and different from 4500, 4789 and 6081.
-
-| `profilePort`
-| `integer`
-| `profilePort` allows setting up a Go pprof profiler listening to this port
+| `multiClusterDeployment`
+| `boolean`
+| Set `multiClusterDeployment` to `true` to enable multi clusters feature. This adds `clusterName` label to flows data
```

Comment:
```suggestion
| Set `multiClusterDeployment` to `true` to enable multi-cluster feature. This adds `clusterName` label to flows data
```

Reply:
According to IBM style guide:
![image](https://github.com/openshift/openshift-docs/assets/77501157/98ea32ab-619c-416b-aa87-6c1ce3465ed8)
<img width="1036" alt="image" src="https://github.com/openshift/openshift-docs/assets/77501157/a906cff2-c91d-49c0-9ab1-8f659f99f192">

I take this guidance, along with a search of "multi-cluster" and "multicluster" in the IBM style guide, to mean that "multicluster" is the correct way to form this word. In my search, I see "multicluster" with no hyphen.

Reply:
@skrthomas you would know better than I! Good with me :+1: